wezterm-gui/src/termwindow/mod.rs

```diff
@@ -212,6 +212,7 @@ pub struct TabInformation {
     pub tab_id: TabId,
     pub tab_index: usize,
     pub is_active: bool,
+    pub is_last: bool,
```

Comment:
let's rename this to something a little more descriptive, because the default connotation of `last` in my native English is "the last one to be enumerated" which implies the right-most tab, but here it is really "the previously active tab" and the ambiguity is potentially confusing!

I realize that https://wezterm.org/config/lua/keyassignment/ActivateLastTab.html sets a precedent for using `last`.

How about `is_last_active` for this and the field method below?